Sources/StreamChatUI/ChatMessageList/Attachments/AttachmentViewCatalog.swift

```diff
@@ -15,8 +15,6 @@ open class AttachmentViewCatalog {
         message: ChatMessage,
         components: Components
     ) -> AttachmentViewInjector.Type? {
-        if message.isDeleted { return nil }
```

Comment:
Why was this check removed?

Reply:
You also commented on this in the other PR 😂 This logic, was moved to a higher level so that this is not overridden by customers. This logic should apply to every injector (including custom ones), and not only our injectors.

Reply:
1. That was last year
2. I'm old 🤣